### PR TITLE
B-02043 Education Messages

### PR DESCRIPTION
**Pull request by robbiecarlton**
All this is missing is the real copy which hasn't been finalized yet. So we can't merge but it's ready for review.

### CODE REVIEW

File: src/holofuel/pages/Inbox/Inbox.js [hunk 1]
```diff
@@ -235,6 +184,11 @@ export default function Inbox ({ history: { push } }) {
         setUserMessage={setUserMessage} />)}
     </div>}
 
+    {inboxView === VIEW.actionable && actionableTransactions.length > 0 && <OneTimeEducationModal
+      id='inbox'
+      message='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.'
```
Comment (robbiecarlton) [inline src/holofuel/pages/Inbox/Inbox.js]: Real copy goes here before merging

File: src/holofuel/pages/TransactionHistory/TransactionHistory.js [hunk 2]
```diff
@@ -117,6 +121,10 @@ export default function TransactionsHistory ({ history: { push } }) {
           partition={partition}
         />)}
     </div>}
+    {shouldShowSentTransactionMessage && <OneTimeEducationModal
+      id='history'
+      message='Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.'
```
Comment (robbiecarlton) [inline src/holofuel/pages/TransactionHistory/TransactionHistory.js]: Real copy goes here before merging

**Review by JettTech — COMMENTED** [review]
I ended up completing this ticket, and have therefore pinged @zo-el for a review.

**Comment by zo-el** [comment]
@JettTech Can I get the scope of this PR?

**Review by zo-el — APPROVED** [review]
LGTM

**Pull request merged**